docs/config/README.md

```diff
@@ -1195,6 +1196,39 @@ truncation_length = 8
 truncation_symbol = '…/'
 ```
 
+## Direnv
+
+The `direnv` module shows the status of the current rc file if one is present. The status includes the path to the rc file, whether it is loaded, and whether it has been allowed by `direnv`.
+
+### Options
+
+| Option         | Default                                  | Description                                 |
+| -------------- | ---------------------------------------- | ------------------------------------------- |
+| `format`       | `'[$loaded/$allowed $rc_path]($style) '` | The format for the module.                  |
+| `style`        | `'bold orange'`                          | The style for the module.                   |
+| `disabled`     | `true`                                   | Disables the `direnv` module.               |
+| `detect_files` | `['.envrc']`                             | Which filenames should trigger this module. |
+
+### Variables
+
+| Variable | Example               | Description                             |
+| -------- | --------------------- | --------------------------------------- |
+| loaded   | `'loaded'`            | Whether the current rc file is loaded.  |
+| allowed  | `'denied'`            | Whether the current rc file is allowed. |
+| rc_path  | `'/home/test/.envrc'` | The current rc file path.               |
+| style\*  | `'red bold'`          | Mirrors the value of option `style`.    |
```

Comment:
The variables in the documentation normally don't use quotation marks.

Reply:
Fixed